docs/MODULES.md

```diff
@@ -88,8 +89,10 @@ This module is used for blending two images .
 ```
 
 where `options` is an object with the following properties:
-* offset: step of image with which current image is to be blended(Two steps back is -2, three                steps back is -3 etc; default -2) 
-* func: function used to blend two images (default : function(r1, g1, b1, a1, r2, g2, b2, a2) {            return [ r1, g2, b2, a2 ] })
+* offset: step of image with which current image is to be blended(Two steps back is -2, three steps back is -3 etc; default -2) 
+* blendMode: Blending mode to use for blending two images by default it uses the given function
```

Comment:
Can we add a lookup table or at least a link to such a table here?

Reply:
Done!